alma/alma-installments-prestashop
# [insurance] Check translation for prestashop insurance

### Reason for change



[Linear task](https://linear.app/almapay/issue/ECOM-1274/check-translation-for-prestashop-insurance)

### Code changes

We updated the translation for insurance feature

### How to test

Add french locale on the prestashop back office and see the french wording for the insurance feature 

### Checklist for authors and reviewers




### Non applicable
